datadog/dogshell/wrap.py

```diff
@@ -239,6 +241,11 @@ def main():
     if notifications:
         event_body.extend([u'notifications: %s\n' % (notifications)])
 
+    if options.tags is not None:
```

Comment:
I don't think `options.tags` can take the `None` value. I would simply check
```python
if  options.tags:
   do_something()
```
to filter out `None`, `False` or `''`